carceral-groups-api/Controllers/GeographicLocationsController.cs

```diff
@@ -20,11 +20,13 @@ public GeographicLocationsController(AppDbContext dbContext)
         {
             var query = _dbContext.Documents.AsQueryable();
 
-            foreach(var item in request.Filters){
-                query = query.Where(m => m.CategoryId == item.CategoryId && m.InstitutionId == item.InstitutionId);
-            }
+            // filed issue https://github.com/DrewAGamboa/Carceral-Groups-Website/issues/93

```

Comment:
@cesarlucatero let me know if you need more info for this. I commented out for now, so that we have geographic locations returning to the map.

Reply:
yea it was doing and AND instead of OR